homeassistant/components/spotify/media_player.py

```diff
@@ -111,6 +109,9 @@ def wrapper(
             if exc.reason == "NO_ACTIVE_DEVICE":
                 raise HomeAssistantError("No active playback device found") from None
             raise HomeAssistantError(f"Spotify error: {exc.reason}") from exc
+        else:
```

Comment:
We don't need an `else:` block. All except cases return or raise.

Reply:
Interesting that pylint didn't catch this.

Reply:
May because the if block is one level deeper

Reply:
Yes, I think pylint doesn't claim this because of the nested if in the except.